mathesar_ui/src/utils/styles.ts

```diff
@@ -0,0 +1,10 @@
+import type { CssVariablesObj } from '@mathesar/types';
+
+export function createStyleString(cssVariables: CssVariablesObj) {
```

Comment:
Can we rename this to something like `makeStyleStringFromCssVariables`, since this is specific only for css variables?

Reply:
a602d28cf